src/tools/blast/components/BlastForm.tsx

```diff
@@ -108,83 +109,98 @@ const BlastForm = () => {
   const dispatch = useDispatch();
   const history = useHistory();
 
+  const getInitialValues = () => {
```

Comment:
this probably should be wrapped in a react `useMemo` function as it's quite heavy logic on each render for something used only once on mount

Reply:
Done